docs/consistency-options.md

```diff
@@ -0,0 +1,69 @@
+# Consistency Options
```

Comment:
Do we think it's possible that this could live as protobuf docstring documentation instead of a dedicated document? The documentation is almost already exactly this already.

Reply:
Its not super discoverable right now, which is why I put it here